### PR TITLE
docs: #1086 add sizing, intro images

### LINKED ISSUE #1086: add sizing, intro images
### Describe the Issue

Add new image 3-node-rp-cluster.png created by Travis/Ben with text clarifying that 225MBps in optimal scenario with multiple consumers and everything well-balanced, but up to 300MBps in the worst-case scenario. 

Also add cluster.png to intro.

### Updates to existing documentation

https://docs.redpanda.com/docs/deploy/deployment-option/self-hosted/manual/sizing-use-cases/#low-throughput-short-retention

https://docs.redpanda.com/docs/get-started/intro-to-events/#what-is-redpanda

### PR DESCRIPTION
**Pull request by micheleRP**
resolves #1086 

adds a new overview image and a new sizing image

preview:
- https://deploy-preview-1110--redpanda-documentation.netlify.app/docs/get-started/intro-to-events/#what-is-redpanda
- https://deploy-preview-1110--redpanda-documentation.netlify.app/docs/deploy/deployment-option/self-hosted/manual/sizing/#network

### CODE REVIEW

**Comment by micheleRP** [comment]
@jrkinley Could you please confirm that you're OK with these 2 new images?

File: docs/get-started/intro-to-events.mdx [hunk 1]
```diff
@@ -26,6 +26,8 @@ Consumers are client applications that subscribe to Redpanda topics to asynchron
 
 Redpanda decouples producers from consumers to allow for asynchronous event processing, event tracking, event manipulation, and event archiving. Producers and consumers interact with Redpanda using the Apache Kafka® API.  
 
+![Producers and consumers in a cluster](../../static/img/cluster.png)
+
```
Comment (jrkinley) [inline docs/get-started/intro-to-events.mdx:30]: If the arrows represent the direction of connections, then this is correct. If the arrows represent data flow then the arrows should point to the consumers. The other image (networking) is showing data flow.

File: docs/deploy/deployment-option/self-hosted/manual/sizing.mdx [hunk 2]
```diff
@@ -28,10 +28,12 @@ In general, choose the number of nodes based on the following criteria, and add
 
 To understand the network usage of a basic Redpanda cluster, consider a cluster with three nodes, a topic with a single partition and a replication factor of three, and a single producer and consumer. For every 75 MB written to the partition’s leader, 150 MB is transmitted across the network to other nodes for replication, and 75 MB is transmitted to the consumer. 
 
-The 150 MB/sec of bandwidth for replication is full duplex (where each byte sent by a broker is received by some other broker). The 75 MB/sec producer and consumer flows, however, are half-duplex, because the client endpoint in each case is outside of the cluster. Therefore, the intra-cluster bandwidth is 225 MB for incoming and outgoing flows:
+![3 node Redpanda cluster](../../../../../static/img/3-node-rp-cluster.png)
+
```
Comment (jrkinley) [inline docs/deploy/deployment-option/self-hosted/manual/sizing.mdx:32]: This image is effectively showing a partition Raft group with a single elected leader and two replicas. Maybe we should highlight that in the image? Or at least label the leader and replicas.

Do we use the cog logo elsewhere in our documentation to illustrate partitions?

Comment (micheleRP) [reply docs/deploy/deployment-option/self-hosted/manual/sizing.mdx:32]: @jrkinley Ben made a separate image. Please confirm if you like this one better, or if you have other suggestions for it!

**Review by jrkinley — APPROVED** [review]

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**